tests/e2e/kubectl.py

```diff
@@ -138,8 +138,10 @@ def create_and_check(manifest, check, ns=None, shell=None, timeout=1800):
     else:
         # Wait for reconcile to start before performing other checks. In some cases it does not start, so we can pass
         # wait_field_changed("chi", chi_name, state_field, prev_state, ns)
-        wait_chi_status(chi_name, "InProgress", ns=ns, retries=3, throw_error=False, shell=shell)
-        wait_chi_status(chi_name, "Completed", ns=ns, shell=shell)
+        # wait_field(kind=kind, name=chi_name, field=".status.status", value="InProgress"
+        #            , ns=ns, retries=3, throw_error=False, shell=shell)
```

Comment:
Why do you remove `InProgress`?

it covers corner cases when nothing happens and chi just not reconciled and not changed